frontend/src/hex/JumpToOffset.svelte

```diff
@@ -56,12 +40,12 @@
     if (e.key === 'Enter') {
       input.blur();
       try {
-        let result = calculator.calculate(input.value) + 1;
-        $scrollY.top = result / dataLength;
+        currentPosition = calculator.calculate(input.value);
```

Comment:
This needs to be rounded. We can now go to an unaligned position like `7` and then the rest of the results will be misaligned when scrolling the hex view.

Also the value no longer clears properly after it has been set. So once it has been set the value no longer updates on scroll.

Last thing is that we should be using the actual value, not the placeholder value. Not only does having full-opacity text look better, but it means that a user can edit the current location instead of having to manually retype it from scratch every time. It would also solve the above problem.


https://github.com/redballoonsecurity/ofrak/assets/99368685/961d85d4-37cc-4d9b-b826-944bdbfcda24

Reply:
The last part of this one is still unresolved – the go to offset text input should set the `value` and not the `placeholder` so that it looks better and stays updated.